x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/middleware/policy_settings_middleware.ts

```diff
@@ -96,7 +96,7 @@ export const policySettingsMiddlewareRunner: MiddlewareRunner = async (
 
     // Agent summary is secondary data, so its ok for it to come after the details
     // page is populated with the main content
-    if (policyItem.policy_id) {
+    if (policyItem.policy_ids.length) {
```

Comment:
does `policy_ids` always exist?

Reply:
Yes, we expect `policy_ids` to be always there (as opposed to depreciated `policy_id`) since it's the only part connecting integration to agent policy.

Reply:
Although we expect policy_ids to always be defined, I noticed some parts of the code where this check is still performed. I’ve added a safe .length check here: f9d85f69dd6346e87b8c2ee62708b2795387593d. Thanks for bringing this to my attention!